### PR TITLE
new techniques: high-throughput scanning & 4-points congruent sets alignment

### PR DESCRIPTION
**Pull request by lzehl**
+ high-throughput scanning
+ 4-points congruent sets alignment

### CODE REVIEW

File: instances/technique/highThroughputScanning.jsonld [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/technique/highThroughputScanning",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
+  "definition": "'High-throughput scanning' is a technique for automatic creation of analog or digital images of a large numbers of samples.",
```
Comment (UlrikeS91) [inline instances/technique/highThroughputScanning.jsonld]: "...of large numbers of samples." OR "...of a large number of samples."

File: instances/technique/highThroughputScanning.jsonld [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  "description": null,
+  "interlexIdentifier": null,
+  "knowledgeSpaceLink": null,
+  "name": "high-throughput scanning",
+  "preferredOntologyIdentifier": null,
+  "synonym": [
+    "high throughput scanning"
```
Comment (UlrikeS91) [inline instances/technique/highThroughputScanning.jsonld:14]: wouldn't abbreviation make sense to add as synonym - HTS? Or is this not commonly used? (not an expert, just asking)

**Review by UlrikeS91 — CHANGES_REQUESTED** [review]

File: instances/technique/highThroughputScanning.jsonld (hunk 2 above)
Comment (lzehl) [reply instances/technique/highThroughputScanning.jsonld:14]: HTS is typically used for high-throughput screening which is a different technique
in principle "high-throughput imaging" is more often used, but implies also image analysis, while the scanning would be analysis free. I'd like to distinguish this (as we do for microscopy techniques)

File: instances/technique/highThroughputScanning.jsonld (hunk 1 above)
Comment (lzehl) [reply instances/technique/highThroughputScanning.jsonld]: corrected

**Review by UlrikeS91 — APPROVED** [review]

**Pull request merged**